frontend/src/components/Tournament.js

```diff
@@ -134,6 +156,26 @@ const Tournament = () => {
           </Match>
         </Switch>
       </div>
+      <Show when={playerTeam()}>
+        <A
+          href={`/tournament/${params.slug}/team/${
+            playerTeam().ultimate_central_slug
+          }`}
+          class="mt-5 block w-full rounded-lg bg-gradient-to-br from-pink-600 to-orange-400 p-0.5 shadow-md"
+        >
+          <div class="rounded-md bg-white p-4 dark:bg-gray-800">
+            <h5 class="mb-2 text-center text-xl tracking-tight">
+              <span class="font-medium">Your team - </span>
+              <span class="font-bold text-gray-600 dark:text-gray-300 ">
```

Comment:
also will it be good to have the same border orange color for team name instead of white?